From 3dea3025c7260bb361047dd4397bb4c12c9c1dec Mon Sep 17 00:00:00 2001
From: Jiri Benc <jbenc@redhat.com>
Date: Wed, 17 Dec 2014 22:12:37 -0500
Subject: [net] ppp: ppp-ioctl.h: pull in ppp_defs.h

Message-id: <f1809ac0a31081598cbee9978822e7d90e8dd32c.1418854351.git.jbenc@redhat.com>
Patchwork-id: 102229
O-Subject: [RHEL7.1 net PATCH] ppp: ppp-ioctl.h: pull in ppp_defs.h
Bugzilla: 1159802
RH-Acked-by: Daniel Borkmann <dborkman@redhat.com>
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Hannes Frederic Sowa <hannes@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1159802
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=8395912

commit e8b671460410c8fd996c8a1c228b718c547cc236
Author: Mike Frysinger <vapier@gentoo.org>
Date:   Thu Jan 23 15:54:09 2014 -0800

    include/uapi/linux/ppp-ioctl.h: pull in ppp_defs.h

    This header uses enum NPmode but doesn't include ppp_defs.h.  If you try
    to use this header w/out including the defs header first, it leads to a
    build failure.  So add the explicit include to fix it.

    Don't know of any packages directly impacted, but noticed while building
    some ppp code by hand.

    Signed-off-by: Mike Frysinger <vapier@gentoo.org>
    Cc: Paul Mackerras <paulus@samba.org>
    Cc: David Miller <davem@davemloft.net>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/uapi/linux/ppp-ioctl.h b/include/uapi/linux/ppp-ioctl.h
index 2d9a885..63a23a3 100644
--- a/include/uapi/linux/ppp-ioctl.h
+++ b/include/uapi/linux/ppp-ioctl.h
@@ -12,6 +12,7 @@
 
 #include <linux/types.h>
 #include <linux/compiler.h>
+#include <linux/ppp_defs.h>
 
 /*
  * Bit definitions for flags argument to PPPIOCGFLAGS/PPPIOCSFLAGS.
-- 
1.7.1